catalog.html

```diff
@@ -174,12 +172,12 @@ <h3 class="hotel-card-title">GreenTree Inn</h3>
         </section>
 
         <section class="subscribe-catalog">
-          <h2 class="title subscribe-title-catalog">Подпишитесь на рассылку</h2>
-          <p class="subtitle subscribe-subtitle-catalog">Только полезная информация и никакого спама, <br>честное бойскаутское!</p>
+          <h2 class="title">Подпишитесь на рассылку</h2>
+          <p class="subtitle subtitle-subscribe subtitle-dark">Только полезная информация и никакого спама, <br>честное бойскаутское!</p>
           <form class="subscribe-form" action="https://echo.htmlacademy.ru/" method="post">
             <label class="visually-hidden" for="subscribe-email">Подписаться.</label>
-            <input class="field" placeholder="Ваш e-mail" type="email" name="subscribe-email" id="subscribe-email" required>
-            <button class="subscribe-button" type="submit">Подписаться</button>
+            <input class="field-subscribe field-background" placeholder="Ваш e-mail" type="email" name="subscribe-email" id="subscribe-email" required>
+            <button class="subscribe-button cyan-button" type="submit"><span class="subscribe-button-text">Подписаться</span></button>
```

Comment:
`span` то зачем здесь? В кнопке есть явный текст, поэтому просто
```suggestion
            <button class="subscribe-button cyan-button" type="submit">Подписаться</button>
```